woof-code/rootfs-skeleton/usr/update-microcode

```diff
@@ -24,10 +24,10 @@
 
   xCPU=""
   
-  [ "$(lscpu | grep "Intel")" != "" ] && xCPU='intel' 
+  [ "$(cat /proc/cpuinfo | grep -m 1 "Intel")" != "" ] && xCPU='intel' 
   
   if [ "$xCPU" == "" ]; then
-   [ "$(lscpu | grep "AMD")" != "" ] && xCPU='amd'   
+   [ "$(cat /proc/cpuinfo | grep -m 1 "AMD")" != "" ] && xCPU='amd'   
```

Comment:
You can even avoid this `cat`: ` grep -m 1 "AMD" /proc/cpuinfo`

Reply:
Of course.